### ISSUE TITLE
Naming conventions

### ISSUE THREAD

**Issue by pmahon3** [pmahon3/USRA2019#5]
Consider: 

[`delta.prime[i] ~ dnorm( muDelta.prime, 1 / sigmaMuDelta.prime^2 )`](https://github.com/pmahon3/UWO-MOTUS-Analysis/blob/332e0f6bdc8e6b29eb94db22c8d1035191061ce0/src/main/R/Population/v1.1/Model.R#L23)

and one of it's hyperpriors,

[`muDelta.prime ~ dnorm( muMuDelta.prime, sigmaMuMuDelta.prime^2)`](https://github.com/pmahon3/UWO-MOTUS-Analysis/blob/332e0f6bdc8e6b29eb94db22c8d1035191061ce0/src/main/R/Population/v1.1/Model.R#L35)

Would it make more sense to say `sigmaDelta.prime` is the standard deviation of `delta.prime` and `sigmaMuDelta.prime` the standard deviation of `muDelta.prime` in the sense that the `sigma` prefix in the name is an operator over the remaining portion of the variable name? Like, 

`delta.prime[i] ~ dnorm( muDelta.prime, 1 / sigmaDelta.prime^2 )`

`muDelta.prime ~ dnorm( muMuDelta.prime, sigmaMuDelta.prime^2)`

**Comment by sjbonner** [comment]
Yes, this would absolutely make sense -- but I still find the naming somewhat hard to follow. I'm going to create a new branch in which I can modify the names in the JAGS code and see if I can come up with anything clearer.

**Comment by sjbonner** [comment]
Hey @pmahon3 . I've updated the file Model.R in the new branch I created. I tried to change the names of the parameters in a way that is consistent and would make it easy to identify what is what. I commited after each change so there should be a trace of exactly what I did, and I also added an explanation in comments at the top.

I did add a little to the model by allowing for the mean to vary by day within individual and period. We should also allow the variance to do the same -- but I need to think about how to model this. 

Let me know what you think!

**Comment by pmahon3** [comment]
Hey Simon, that looks good to me, the specified indexing really helps as well. I'll close this issue and merge these changes into master. I believe there is one additional area where we could add variation, I'll pick up that conversation on the other issue.
